src/m365/spo/commands/site/SiteProperties.ts

```diff
@@ -2,4 +2,26 @@ export interface SiteProperties {
   Status: string;
   Title: string;
   Url: string;
-}
\ No newline at end of file
+}
+
+export interface RecycleBinItemProperties{
```

Comment:
Since we're not using this type for anything, we don't need to define it. We could define the request's response type as `any[]`, because we pass the output as-is to the logger so having this type doesn't add any value.